### ISSUE TITLE
ARIA hidden element must not contain focusable elements - tabindex -1 on mobile when hidden

### ISSUE THREAD

**Issue by stef-barburou** [snap-framework/snap#20]
Language: English

Tool(s): aXe

Browser/Screen reader: Firefox

WCAG 2.0 reference: 4.1.2 Name, role, value

Details: aXe reported that: ARIA hidden element must not contain focusable elements
Issue description: Ensures aria-hidden elements do not contain focusable elements

ARIA hidden element must not contain focusable elements. This is module-wide

Using the aria-hidden="true" attribute on an element removes the element and ALL of its child nodes from the accessibility API making it completely inaccessible to screen readers and other assistive technologies. Aria-hidden may be used with extreme caution to hide visibly rendered content from assistive technologies only if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. If aria-hidden is used to hide visible content from screen readers, the identical or equivalent meaning and functionality must be exposed to assistive technologies.

Note: Using aria-hidden="false" on content that is a descendent of an element that is hidden using aria-hidden="true" will NOT expose that content to the accessibility API and it will not be accessible to screen readers or other assistive technologies.

The rule applies to any element with an aria-hidden="true" attribute.

By adding aria-hidden="true" to an element, content authors ensure that assistive technologies will ignore the element. This can be used to hide decorative parts of a web page, such as icon fonts - that are not meant to be read by assistive technologies.

A focusable element with aria-hidden="true" is ignored as part of the reading order, but still part of the focus order, making it’s state of visible or hidden unclear.

·       https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

·       https://www.w3.org/TR/wai-aria-1.1/#aria-hidden

·       https://www.w3.org/TR/html/editing.html#can-be-focused

Element location:

#mb-pnl

Element source:

<div id="mb-pnl" class="wb-overlay modal-content overlay-def wb-panel-r wb-overlay-inited user-closed" aria-hidden="true">

To solve this violation, you need to:

Fix the following:

·       Focusable content should have tabindex='-1' or be removed from the DOM
